.buildkite/pipelines/on_merge_unsupported_ftrs.yml

```diff
@@ -65,11 +84,35 @@ steps:
   - command: .buildkite/scripts/steps/functional/defend_workflows.sh
     label: 'Defend Workflows Cypress Tests'
     agents:
-      queue: n2-4-virt
+      image: family/kibana-ubuntu-2004
+      imageProject: elastic-images-qa
+      provider: gcp
+      enableNestedVirtualization: true
+      localSsdInterface: nvme
+      localSsds: 1
+      machineType: n2-standard-4
     depends_on: build
     timeout_in_minutes: 60
     parallelism: 10
+    retry:
+      automatic:
+        - exit_status: '-1'
+          limit: 1
+
+  - command: .buildkite/scripts/steps/functional/defend_workflows_serverless.sh
```

Comment:
This step was added, and not sure if that was intended. Maybe it was not in 8.13 for a reason?

Reply:
Good find! This is probably an unwanted guest through backport - I'll remove it